maintenance/main.py

```diff
@@ -158,7 +158,10 @@ def black_format(code):
 
 
 def ensembl_stdpopsim_id(ensembl_id):
+    if ensembl_id == "canis_lupus_familiaris":
+        ensembl_id = "canis_familiaris"
```

Comment:
this is a horrible, terrible thing that i've done, but i've done it because this single ensembl shift disagrees with all the other ensembl ids....